src/math/Box2.js

```diff
@@ -129,27 +129,13 @@ Box2.prototype = {
 
 	containsPoint: function ( point ) {
 
-		if ( point.x < this.min.x || point.x > this.max.x ||
```

Comment:
Well... In this case, it makes the code less readable. And Closure Compiler already simplifies this when minifying to `three.min.js`.

Also, interestingly, Closure compiler produces this instead:

```js
return box.max.x < this.min.x || box.min.x > this.max.x || box.max.y < this.min.y || box.min.y > this.max.y || box.max.z < this.min.z || box.min.z > this.max.z ? false : true;
```

Which I think it's more readable.